go/systests/teams_test.go

```diff
@@ -665,7 +665,7 @@ func (u *userPlusDevice) provisionNewDevice() *deviceWrapper {
 	require.NotNil(t, u.backupKey, "Add user with paper key to use provisionNewDevice")
```

Comment:
Changes in this file were because this `NotNil` check was silently always passing on the object that wasn't a pointer, even when it wasn't initialized.